_pages/dev/features/reorder.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Reorder
+---
+
+The reorder feature allows you to create cart from an existing order.
```

Comment:
Overall, released documentation for similar features provide more details about what the feature does, its behaviour, requirements and relevant limitations.  There is also detailed info about what impel to use to add the required CMS component that will enable the feature.

You can get examples from [Saved Carts](https://help.sap.com/docs/SAP_COMMERCE_COMPOSABLE_STOREFRONT/eaef8c61b6d9477daf75bff9ac1b7eb4/af7716ec6b264f328f99bfa684577ff1.html?locale=en-US), or [Quick Order](https://help.sap.com/docs/SAP_COMMERCE_COMPOSABLE_STOREFRONT/eaef8c61b6d9477daf75bff9ac1b7eb4/d43711ecc5c74bc2b00024ba2f8ad6b8.html?locale=en-US)

Reply:
There is no impex and I believe most of the other information is already in the doc.

Reply:
It is a core feature so there isn't any impex. Behaviour is "creating cart from an existing order" and how to is also mentioned. Also mentioned what CMS component is involved. I do not know of any limitations though but from understanding there doesn't seem to be any.

Reply:
The context of those docs is: many customers don't start their project from the sample data.  They start with and empty CMS and add the things they need for their specific purpose.  This is why I presume the docs for saved carts and quick order document what impex can add the required cms components to enable the feature.